catt/controllers.py

```diff
@@ -54,15 +86,23 @@ def clear(self):
 
 
 class CastController:
-    def __init__(self):
+    def __init__(self, device_name):
         cache = Cache()
+        cached_ip = cache.get(device_name)
```

Comment:
I mean, if my Chromecast is called "Stavros' Chromecast", can I specify the name as "stavros" and still have it match?